etp-front/src/components/H/H2.svelte

```diff
@@ -8,4 +9,4 @@
   }
 </style>
 
-<h2>{text}</h2>
+<h2 {name} data-cy={name}>{text}</h2>
```

Comment:
Of `data-cy` it is quite obvious that it is related to Cypress, but is that `name` attribute also intentional?